src/applications/gi/selectors/calculator.js

```diff
@@ -795,10 +807,13 @@ export const getCalculatedBenefits = createSelector(
     const giBillChapter = +eligibility.giBillChapter;
     const institutionType = institution.type.toLowerCase();
     const isOJT = institutionType === 'ojt';
+    const beneficiaryLocationQuestion = giBillChapter === 33;
 
     calculatedBenefits.inputs = {
       inState: false,
       tuition: true,
+      // only necessay for chapter 33 recipients who are the only beneficiaries to receive a housing allowance (BAH)
```

Comment:
trying to add some comments on the things I understand- we don't have any test scenarios from our stakeholders on this calculation.